draft-ietf-rats-corim.md

```diff
@@ -1103,15 +1103,9 @@ Environment.
 
 #### Endorsed Values Triple {#sec-comid-triple-endval}
 
-An Endorsed Values triple declares additional measurements that are valid when
-a Target Environment has been verified against reference measurements. For
-Endorsed Value Claims, the subject is either a Target or Attesting Environment,
-the object contains measurements, and the predicate defines semantics for how
-the object relates to the subject.
-
-Said another way, take the pair of `environment-map` and `measurement-map / mkey` from a `endorsed-triple-record`.
-If this same pair matches the pair from a `reference-triple-record` from the same CoMID, consider that record to be an implicit condition for adding the `endorsed-triple-record` to the ACS.
-<!-- Issue#251: is this also an implicit condition for the other endorsement triples? -->
+An Endorsed Values triple declares additional measurements to add to the ACS when its environment has any other existing measurements.
```

Comment:
The existence of measurement is not a requirement.  After all, the matching is done only on the environment.

One could imagine the ACS loaded with environment-only evidence (i.e., one which doesn't carry any measurement, only signed environment identifier(s)).  If the environment in the endorsed value triple matches, the associated measurement (really, a property) is added to the ACS - e.g., "the chipset is enzyme-based" - a bio-RoT :-)